### PR TITLE
[ORCA-114] Created Upgrading ORCA page and added content.

### PR DESCRIPTION
**Pull request by Quarriers89**
## Summary of Changes

Added a page and content for Upgrading ORCA.

Addresses [ORCA-114: Create "Upgrading ORCA" page for ORCA developer documentation.](https://bugs.earthdata.nasa.gov/browse/ORCA-114)

## Changes

* Made a new page for Upgrading ORCA
* ...

## Type of change

Please delete options that are not relevant.

- [X] This change requires a documentation update

## How Has This Been Tested?

N/A

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [X] User documentation

## What is the current behavior?

N/A

## What is the expected correct/added behavior?

N/A

## Relevant logs and/or screenshots

N/A

## Possible fixes

N/A

### CODE REVIEW

File: website/sidebars.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ module.exports = {
         'developer/deployment-guide/deployment-environment',
         'developer/deployment-guide/deployment-s3-bucket',
         'developer/deployment-guide/deployment-with-cumulus',
+        'developer/deployment-guide/upgrading-orca'
```
Comment (bhazuka) [inline website/sidebars.js:50]: path should be `'developer/deployment-guide/deployment-upgrading-orca'` based on your page `id`.

File: website/docs/developer/deployment-guide/deployment-upgrading-orca.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+title: Upgrading ORCA
+description: Provides developer with information on upgrading ORCA.
+---
+
+After the initial ORCA deployment, any future updates to the ORCA deployment 
+from configuration files, Terraform files (`*.tf`), or modules from a new 
+version of ORCA can be deployed and will update the appropriate portions of 
+the stack as needed.
+
+**IMPORTANT:**  For instances where ORCA has been deployed with Cumulus,
```
Comment (bhazuka) [inline website/docs/developer/deployment-guide/deployment-upgrading-orca.md:12]: Remove `For instances where ORCA has been deployed with Cumulus` we will always be deployed with Cumulus for the upcoming future.

Change `**IMPORTANT**` to a [admonition](https://nasa.github.io/cumulus-orca/docs/developer/development-guide/documentation/documentation-style-guide#admonitions) `:::important Ensure Cumulus Compatibility` see the linked style guide for syntax.

File: website/docs/developer/deployment-guide/deployment-upgrading-orca.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+visit the [Supported PI Versions wiki page](https://wiki.earthdata.nasa.gov/display/CUMULUS/Supported+PI+Versions) 
+to ensure version compatibility.
+
+## ORCA Versioning
+
+ORCA uses a global versioning approach, meaning version numbers are 
+consistent across all Terraform modules and semantic versioning to track 
+major, minor, and patch version (e.g., 1.0.0).
+
+**IMPORTANT:** By convention, ORCA minor version releases introduce breaking
```
Comment (bhazuka) [inline website/docs/developer/deployment-guide/deployment-upgrading-orca.md:22]: Change this **IMPORTANT** block to an admonition.

Change `ORCA minor version releases introduce breaking changes (e.g., 1.13.x -> 1.14.0), so it is critical that the release notes are viewed for migration steps`
to something like `ORCA major version releases (e.g., 2.x.x -> 3.x.x) introduce known breaking changes. However, any version change has the possibility to introduce breaking changes for your particular use case. It is critical that  the release notes are viewed for migration steps and changes.`

File: website/docs/developer/deployment-guide/deployment-upgrading-orca.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+To update your ORCA version:
+
+1.	Find the desired release on the [ORCA Releases page](https://github.com/nasa/cumulus-orca/releases).
+
+2.	Update the `source` in your Terraform deployment files for ORCA by 
+replacing `vx.x.x` with the desired version of ORCA.
+
+3.	Run `terraform init` to get the latest copies of your updated modules.
+
+## Update Data Persistence Resources
```
Comment (bhazuka) [inline website/docs/developer/deployment-guide/deployment-upgrading-orca.md:78]: Remove `## Update Data Persistence Resources` section. This is for Cumulus only

File: website/docs/developer/deployment-guide/deployment-upgrading-orca.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+if necessary. From the directory of your `data-persistence` deployment module 
+(e.g., `data-persistence-tf`):
+
+`$ AWS_REGION=<region> \ # e.g. us-east-1`
+    `AWS_PROFILE=<profile> \`
+    `terraform apply`
+
+## Update ORCA Resources
+
+**Reminder:** Remember to [initialize Terraform](https://nasa.github.io/cumulus/docs/deployment/deployment-readme#initialize-terraform)
```
Comment (bhazuka) [inline website/docs/developer/deployment-guide/deployment-upgrading-orca.md:90]: Change `**Reminder**` to an admonition

File: website/docs/developer/deployment-guide/deployment-upgrading-orca.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+if necessary. From the directory of your `cumulus` deployment module (e.g., 
+`cumulus-tf`):
+
+`$ AWS_REGION=<region> \ # e.g. us-east-1`
```
Comment (bhazuka) [inline website/docs/developer/deployment-guide/deployment-upgrading-orca.md:94]: change `us-east-1` to `us-west-2`

**Review by bhazuka — CHANGES_REQUESTED** [review]
A few minor formatting items. Some minor verbiage changes/suggestions. Overall looks good.

File: website/sidebars.js (hunk 1 above)
Comment (Quarriers89) [reply website/sidebars.js:50]: made update to sidebars.js.

File: website/docs/developer/deployment-guide/deployment-upgrading-orca.md (hunk 2 above)
Comment (Quarriers89) [reply website/docs/developer/deployment-guide/deployment-upgrading-orca.md:12]: completed admonition

File: website/docs/developer/deployment-guide/deployment-upgrading-orca.md (hunk 3 above)
Comment (Quarriers89) [reply website/docs/developer/deployment-guide/deployment-upgrading-orca.md:22]: made update to add the admonition

File: website/docs/developer/deployment-guide/deployment-upgrading-orca.md (hunk 4 above)
Comment (Quarriers89) [reply website/docs/developer/deployment-guide/deployment-upgrading-orca.md:78]: done

File: website/docs/developer/deployment-guide/deployment-upgrading-orca.md (hunk 5 above)
Comment (Quarriers89) [reply website/docs/developer/deployment-guide/deployment-upgrading-orca.md:90]: updated to an admonition

File: website/docs/developer/deployment-guide/deployment-upgrading-orca.md (hunk 6 above)
Comment (Quarriers89) [reply website/docs/developer/deployment-guide/deployment-upgrading-orca.md:94]: made update to us-west-2

**Review by bhazuka — APPROVED** [review]

**Pull request merged**